docs/content/getting-started.md

```diff
@@ -95,8 +95,14 @@ A component's template can have "view ports," which are holes in the DOM for loa
 A component's router tells the component what to put inside the view ports based on URL.
 The configuration maps routes to components for each view port.
 
-Let's male a `home` component that our app can route to. First, run `mkdir -p components/home` to make a few directories.
-Then `touch components/home/home.html` and `touch components/home/home.js`.
+Let's make a `home` component that our app can route to.
+
+```
+mkdir -p components/home
+touch components/home/home.html components/home/home.js
+```
+
+This creates our component directory and it's corresponding files, which is a template and a JavaScript component.
```

Comment:
I've introduced proper syntax highlighting here, since it's used later for the `details` component too.